RELEASE-NOTES.txt

```diff
@@ -1,3 +1,7 @@
+21.8
+-----
+* [*] Add ability to block a followed site. [#20053]
```

Comment:
Thanks for adding this release note @salimbraksa 👋 

I was thinking if we should add `Reader:` as a prefix, to make it more explicit where this new functionality was added. Wdyt?

Reply:
@salimbraksa Thanks for applying the suggestion to improve the Release notes in commit [df0d1e3](https://github.com/wordpress-mobile/WordPress-iOS/pull/20053/commits/df0d1e3a8020a98cc8f867cfeaebaa41c56f41a9) 🙇